project1-sps/script.js

```diff
@@ -67,21 +57,17 @@ var main = function (input) {
   console.log(`The number of wins is ${win}`);
   console.log(`The number of draws is ${draw}`);
   console.log(`The number of losses is ${lose}`);
-   
-  return result;
-
 
+  return result;
 };
 
-var rollDice = function(){
+var rollDice = function () {
   var randomDecimal = Math.random() * 3;
   var randomInteger = Math.floor(randomDecimal);
   var diceNumber = randomInteger + 1;
   return diceNumber;
-}
-
-
+};
```

Comment:
helper functions should be declared above the main function or declared before using them